changelog.d/13372.misc

```diff
@@ -0,0 +1 @@
+Make docker images build on armv7 by installing cryptography dependencies in the "requirements" stage. Contributed by Jasper Spaans.
```

Comment:
As per https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#changelog, changes to the Docker image should use the `.docker` extension.

Reply:
Ok - note that I did put it in misc because it changes only an intermediate image, not the deliverable. Will update.